src/qml/WebCatalogPage.qml

```diff
@@ -170,11 +170,7 @@ Page {
 
             onClicked: {
                 pageStack.push(Qt.resolvedUrl("WebPatchPage.qml"),
-                               {
-                                   modelData: model,
-                                   delegate: background,
-                                   versions: versions
-                               })
+                               { modelData: model, versions: versions })
```

Comment:
so delegate was not used here?

Reply:
No, it is/was used in other *PatchPage pages (which are now UnifiedPatchPage). WebPatchPage does not have or use the property, and spat a warning message. (See other comment for the log.)